deep-learning/README.md

```diff
@@ -18,6 +18,16 @@ In these courses, you will learn the foundations of Deep Learning, understand ho
 1. [Introduction to Computer Vision](https://www.udacity.com/course/introduction-to-computer-vision--ud810)
 2. [CS231n: Convolutional Neural Networks for Visual Recognition](http://cs231n.stanford.edu/)
 
+### Roadmap to Deep Learning with Images
+1. Get started with [Introduction to Computer Vision](https://www.udacity.com/course/introduction-to-computer-vision--ud810) to get an idea of how Image Processing and Computer Vision works. You can do this course upto *Lesson 3: Linearity and Convolution* before moving to DL. This builds a strong mathematical foundation and helps you to easily grasp DL later.
```

Comment:
See, if one's focus is to use deep learning for object detection, segmentation (feature level understanding) etc, then understanding only Image Processing is sufficient.  [Digital image processing by Rafael C. Gonzalez](http://web.ipac.caltech.edu/staff/fmasci/home/astro_refs/Digital_Image_Processing_2ndEd.pdf) is a standard book to refer. If one's focusing on deep learning for Computer Vision/Perception application then Image Processing and complete course on Introduction to Computer Vision is requisite. Perception includes concepts of 3D perspective, Stereo, Optical flow, object tracking, visual recognition etc which are all very important. In simple words, Image Processing is kinematics and Computer Vision is dynamics.